### PR TITLE
Desktop Web: fix separator of MD page is super white in the dark theme

### LINKED ISSUE #305: Desktop Web: separator of MD page is super white in the dark theme
Combination: dark theme + desktop + web (FF and chrome)

grafik

Does not happen in the compact layout 🤷

### PR DESCRIPTION
**Pull request by Feichtmeier**
grafik

Fixes #305

## Pull request checklist

- [ ] Either this PR does not introduce visual changes, or
- [X] I run `flutter test --update-goldens` and committed the changes if there were any, or
- [ ] I added before/after/light/dark screenshots if the visual changes I made were not covered by golden tests.

### CODE REVIEW

**Comment by Jupi007** [comment]
Thanks for working on this.
Are your change supporting rtl?

**Comment by Feichtmeier** [comment]
ah no rtl
but i found a better solution I think

those goldy 🏅 tests - do I need to commit them all, or only the PNGs for the files I touched?

File: lib/src/layouts/yaru_landscape_layout.dart [hunk 1]
```diff
@@ -163,6 +164,15 @@ class _YaruLandscapeLayoutState extends State<YaruLandscapeLayout> {
   }
 
   Widget _buildVerticalSeparator() {
+    // Fix for the ultra white divider on flutter web
+    final darkAndWeb =
+        Theme.of(context).brightness == Brightness.dark && kIsWeb;
```
Comment (jpnurmi) [inline lib/src/layouts/yaru_landscape_layout.dart:169]: if you swap the conditions around, we avoid an unnecessary inherited theme lookup on non-web platforms
```
kIsWeb && Theme.of(context).brightness == Brightness.dark
```

**Comment by Feichtmeier** [comment]
@.@

00:14 +2: /home/runner/work/yaru_widgets.dart/yaru_widgets.dart/test/controls/yaru_circular_progress_indicator_test.dart: ... (variant: indeterminate-light: themeMode: ThemeMode.light, value: null)  

??? what's this fun again

**Comment by jpnurmi** [comment]
Your images are looking strange and pixelated. :eyes: Did you switch back to the stable channel after trying the master channel badges? :slightly_smiling_face:

**Comment by jpnurmi** [comment]
Or perhaps the problem is on your side because neither the CI nor my system agrees with these golden changes...? :P

**Comment by Feichtmeier** [comment]
Never tried those tests before
Don't have time for this now
Was 10 lines of code and the screenshot
And the build runs
I'll try to see if I have time and nerves for this later

**Pull request closed without merging**